spec/controllers/archival_media_collections_controller_spec.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+require 'rails_helper'
+
+RSpec.describe ArchivalMediaCollectionsController do
+  let(:user) { nil }
+  let(:adapter) { Valkyrie::MetadataAdapter.find(:indexing_persister) }
+  let(:persister) { adapter.persister }
+  let(:query_service) { adapter.query_service }
+  before do
+    sign_in user if user
+  end
+
+  context "when an admin" do
+    let(:user) { FactoryBot.create(:admin) }
+
+    describe "GET /collections/new" do
+      render_views
+      # This creates a Collection for now; this should change as part of work for #1006
```

Comment:
I'm not sure I understand why this has to create a Collection — can't ArchivalMediaCollectionsController set the resource class to create an ArchivalMediaCollection instead?

Reply:
Only because I didn't know to do that. i'll upate.